services/tenant-management-service/src/webhook.component.ts

```diff
@@ -126,12 +137,24 @@ export class WebhookTenantManagementServiceComponent implements Component {
       TenantOnboardDTO,
       VerifyLeadResponseDTO,
       WebhookDTO,
+      TenantConfig,
     ];
 
-    this.controllers = [WebhookController];
+    this.controllers = [
+      WebhookController,
+      IdpController,
+      TenantConfigController,
+      TenantConfigTenantController,
```

Comment:
remove tenantConfigController and tenantconfigtenat controller from here. only idp controller should be her